### ISSUE TITLE
Unable to click on previous/next month buttons when contains child element

### ISSUE THREAD

**Issue by harrington101** [wakirin/Litepicker#197]
**Describe the bug**
When a child element is set using the buttonText option for the next month/previous months the child element is not clickable. This is necessary to insert font awesome or other icons. 

**To Reproduce**
1. Create a date picker with the following passed: 
```
buttonText: {
            previousMonth: "<i>X</i>",
            nextMonth: "<i class="fas fa-angle-right"></i>"
}
```
2. Clicking in the middle has no effect.
3. Clicking on the very edge of the button works. 

**Expected behavior**
Should be able to click the icon and move to the next/previous month.

**Comment by wakirin** [comment]
fixed in 2.0.9

**Comment by harrington101** [comment]
Thanks. I just pulled it and it is working now.
